api/descriptions/tags/tag_cve_post.md

```diff
@@ -1,5 +1,8 @@
 Sets a tag to a vulnerability based on Common Vulnerability and Exposures (CVE) ID, package, and resource.
-When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
+
```

Comment:
@MorShabiPAN : Can you please review the note?

Reply:
looks good